spec/core_functions/color/hsl/error/five_args.hrx

```diff
@@ -1,19 +1,19 @@
-<===> input.scss
-five-arg {
-  x: hsl(0, 100%, 50%, 0.5, 0);
+<===> error/too_many_args/input.scss
```

Comment:
These are already in an error directory (spec/core_functions/color/hsl/error) so they don't need an additional error component here. I think it's fine to also leave off too_many_args, since there's only one meaningful test for passing five arguments anyway.

Reply:
Good points, fixed.